python/paddle/nn/functional/activation.py

```diff
@@ -136,7 +136,7 @@ def elu(x, alpha=1.0, name=None):
 def elu_(x, alpha=1.0, name=None):
     r"""
     Inplace version of ``elu`` API, the output Tensor will be inplaced with input ``x``.
-    Please refer to :ref:`api_nn_cn_elu`.
+    Please refer to :ref:`api_paddle_nn_ELU`.
```

Comment:
用 functional 下的 `elu` 比较合适。
```suggestion
    Please refer to :ref:`api_paddle_nn_functional_elu`.
```

Reply:
Down